### PR TITLE
Draftcoin 1.0.0.3

### PR DESCRIPTION
**Pull request by tiranad**
Updates to copyright.
Updates for checkpoints >25000
Updates for masternode addresses
Includes contrib/installer  linux auto installer/compiler

### CODE REVIEW

File: .cproject [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (daviunx) [inline .cproject]: This comes from IDE or something like need to remove

File: Makefile [hunk 2]
```diff
@@ -16,7 +16,7 @@ CXXFLAGS      = -fpermissive -fstack-protector-all --param ssp-buffer-size=1 -O2
 INCPATH       = -I/usr/share/qt4/mkspecs/linux-g++-64 -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtNetwork -I/usr/include/qt4/QtGui -I/usr/include/qt4 -Isrc -Isrc/json -Isrc/qt -Isrc/leveldb/include -Isrc/leveldb/helpers -Ibuild -Ibuild
 LINK          = g++
 LFLAGS        = -m64 -fstack-protector-all --param ssp-buffer-size=1 -Wl,-O1
-LIBS          = $(SUBLIBS)  -L/usr/lib/x86_64-linux-gnu -lminiupnpc /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libmemenv.a -lssl -lcrypto -ldb_cxx -lboost_system -lboost_filesystem -lboost_program_options -lboost_thread -lrt -ldl -lpthread -lQtGui -lQtNetwork -lQtCore 
+LIBS          = $(SUBLIBS)  -L/usr/lib/x86_64-linux-gnu -lminiupnpc /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libmemenv.a -lssl -lcrypto -ldb_cxx -lboost_system -lboost_filesystem -lboost_program_options -lboost_thread -lrt -ldl -lpthread -lQtGui -lQtNetwork -lQtCore
```
Comment (daviunx) [inline Makefile]: checkout path names are referencing your home directory

File: Makefile [hunk 3]
```diff
@@ -318,7 +318,7 @@ first: all
 
 all: Makefile $(TARGET)
 
-$(TARGET): /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/drafcoin.03/build/build.h build/ui_coincontroldialog.h build/ui_sendcoinsdialog.h build/ui_addressbookpage.h build/ui_signverifymessagedialog.h build/ui_aboutdialog.h build/ui_editaddressdialog.h build/ui_transactiondescdialog.h build/ui_overviewpage.h build/ui_sendcoinsentry.h build/ui_askpassphrasedialog.h build/ui_rpcconsole.h build/ui_optionsdialog.h $(OBJECTS)  
+$(TARGET): /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/build/build.h build/ui_coincontroldialog.h build/ui_sendcoinsdialog.h build/ui_addressbookpage.h build/ui_signverifymessagedialog.h build/ui_aboutdialog.h build/ui_editaddressdialog.h build/ui_transactiondescdialog.h build/ui_overviewpage.h build/ui_sendcoinsentry.h build/ui_askpassphrasedialog.h build/ui_rpcconsole.h build/ui_optionsdialog.h $(OBJECTS)
```
Comment (daviunx) [inline Makefile]: same here referencing you local home

File: Makefile [hunk 4]
```diff
@@ -384,7 +384,7 @@ dist:
 
 clean:compiler_clean 
 	-$(DEL_FILE) $(OBJECTS)
-	-$(DEL_FILE) /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a; cd /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb ; $(MAKE) clean
+	-$(DEL_FILE) /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a; cd /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb ; $(MAKE) clean
```
Comment (daviunx) [inline Makefile]: same here referencing you local home

File: Makefile [hunk 5]
```diff
@@ -395,11 +395,11 @@ distclean: clean
 	-$(DEL_FILE) Makefile
 
 
-/home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a: FORCE
-	cd /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb && CC=gcc CXX=g++ $(MAKE) OPT="-fpermissive -fstack-protector-all --param ssp-buffer-size=1 -O2" libleveldb.a libmemenv.a
+/home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a: FORCE
```
Comment (daviunx) [inline Makefile]: same here referencing you local home

File: src/makefile.osx [hunk 6]
```diff
@@ -6,8 +6,8 @@
 # Mac OS X makefile for DraftCoin
 # Originally by Laszlo Hanyecz (solar@heliacal.net)
 
-CXX=llvm-g++
-DEPSDIR=/opt/local
+CXX=g++
+DEPSDIR=/opt/osx
```
Comment (daviunx) [inline src/makefile.osx]: I think this is just for testing need to update before merging

**Review by daviunx — CHANGES_REQUESTED** [review]
Looks good i will proceed with testing :)

**Comment by tiranad** [comment]
Just needs a qmake clean. Sorry.On Jan 31, 2018 11:21 AM, daviunx <notifications@github.com> wrote:@daviunx requested changes on this pull request.

Looks good i will proceed with testing :)

In .cproject:
> @@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

This comes from IDE or something like need to remove



In Makefile:
> @@ -16,7 +16,7 @@ CXXFLAGS      = -fpermissive -fstack-protector-all --param ssp-buffer-size=1 -O2
 INCPATH       = -I/usr/share/qt4/mkspecs/linux-g++-64 -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtNetwork -I/usr/include/qt4/QtGui -I/usr/include/qt4 -Isrc -Isrc/json -Isrc/qt -Isrc/leveldb/include -Isrc/leveldb/helpers -Ibuild -Ibuild
 LINK          = g++
 LFLAGS        = -m64 -fstack-protector-all --param ssp-buffer-size=1 -Wl,-O1
-LIBS          = $(SUBLIBS)  -L/usr/lib/x86_64-linux-gnu -lminiupnpc /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libmemenv.a -lssl -lcrypto -ldb_cxx -lboost_system -lboost_filesystem -lboost_program_options -lboost_thread -lrt -ldl -lpthread -lQtGui -lQtNetwork -lQtCore 
+LIBS          = $(SUBLIBS)  -L/usr/lib/x86_64-linux-gnu -lminiupnpc /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libmemenv.a -lssl -lcrypto -ldb_cxx -lboost_system -lboost_filesystem -lboost_program_options -lboost_thread -lrt -ldl -lpthread -lQtGui -lQtNetwork -lQtCore 

checkout path names are referencing your home directory



In Makefile:
> @@ -318,7 +318,7 @@ first: all
 
 all: Makefile $(TARGET)
 
-$(TARGET): /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/drafcoin.03/build/build.h build/ui_coincontroldialog.h build/ui_sendcoinsdialog.h build/ui_addressbookpage.h build/ui_signverifymessagedialog.h build/ui_aboutdialog.h build/ui_editaddressdialog.h build/ui_transactiondescdialog.h build/ui_overviewpage.h build/ui_sendcoinsentry.h build/ui_askpassphrasedialog.h build/ui_rpcconsole.h build/ui_optionsdialog.h $(OBJECTS)  
+$(TARGET): /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/build/build.h build/ui_coincontroldialog.h build/ui_sendcoinsdialog.h build/ui_addressbookpage.h build/ui_signverifymessagedialog.h build/ui_aboutdialog.h build/ui_editaddressdialog.h build/ui_transactiondescdialog.h build/ui_overviewpage.h build/ui_sendcoinsentry.h build/ui_askpassphrasedialog.h build/ui_rpcconsole.h build/ui_optionsdialog.h $(OBJECTS)  

same here referencing you local home



In Makefile:
> @@ -384,7 +384,7 @@ dist:
 
 clean:compiler_clean 
 	-$(DEL_FILE) $(OBJECTS)
-	-$(DEL_FILE) /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a; cd /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb ; $(MAKE) clean
+	-$(DEL_FILE) /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a; cd /home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb ; $(MAKE) clean

same here referencing you local home



In Makefile:
> @@ -395,11 +395,11 @@ distclean: clean
 	-$(DEL_FILE) Makefile
 
 
-/home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb/libleveldb.a: FORCE
-	cd /home/jcoberly/Documents/Wallets/drafcoin.03/src/leveldb && CC=gcc CXX=g++ $(MAKE) OPT="-fpermissive -fstack-protector-all --param ssp-buffer-size=1 -O2" libleveldb.a libmemenv.a
+/home/jcoberly/Documents/Wallets/DraftCoin/drafcoin.03/src/leveldb/libleveldb.a: FORCE

same here referencing you local home



In src/makefile.osx:
> @@ -6,8 +6,8 @@
 # Mac OS X makefile for DraftCoin
 # Originally by Laszlo Hanyecz (solar@heliacal.net)
 
-CXX=llvm-g++
-DEPSDIR=/opt/local
+CXX=g++
+DEPSDIR=/opt/osx

I think this is just for testing need to update before merging

**Pull request merged**